lib/osv/types.py

```diff
@@ -139,6 +141,13 @@ class CommitRange(ndb.Model):
   fixed_in = ndb.StringProperty()
 
 
+class SourceOfTruth(enum.IntEnum):
+  """Source of truth."""
+  NONE = 0
+  INTERNAL = 1
```

Comment:
Do you want to comment to explain what is difference b/w internal and source repo ?

Reply:
Done!